File: support_tools/swigwin-3.0.12/Examples/test-suite/go/typedef_scope_runme.go
package main

import "./typedef_scope"

func main() {
	b := typedef_scope.NewBar()
	x := b.Test1(42, "hello")
	if x != 42 {
		panic("Failed!!")
	}

	xb := b.Test2(42, "hello")
	if xb != "hello" {
		panic("Failed!!")
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/typedef_scope_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'typedef_scope'

b = Typedef_scope::Bar.new

x = b.test1(42, "hello")
if x != 42
  puts "Failed!!"
end

x = b.test2(42, "hello")
if x != "hello"
  puts "Failed!!"
end
